app/models/concerns/recurring_logic_extensions.rb

```diff
@@ -1,9 +1,7 @@
-module Concerns
```

Comment:
This is not necessary if you remove `config.autoload_paths += Dir["#{config.root}/app/models/concerns"]` in `engine.rb`

Reply:
Thanks @ofedoren for looking into this. I guess, I can remove the `concern` paths from config.autoload_path, though. At least, that is what documentation says... https://guides.rubyonrails.org/classic_to_zeitwerk_howto.html#concerns

Reply:
Not really, it's one of the "magic" directories in Rails. `app/models/concerns` is treated as one of "root" directories for Zeitwerk.